src/app/browse-page/my-collections.component.ts

```diff
@@ -1,35 +1,28 @@
 import { Component, OnInit } from '@angular/core'
 import { Router, ActivatedRoute, Params } from '@angular/router'
-import { Subscription }   from 'rxjs'
+import { Subscription } from 'rxjs'
 import { map } from 'rxjs/operators'
+import { HttpClient } from '@angular/common/http'
 
-import { TagsService } from './tags.service'
-import { Tag } from './tag/tag.class'
+import { LockerService } from 'app/_services'
 import { TitleService, AssetSearchService, AuthService, AssetService, FlagService } from '../shared'
 
 @Component({
   selector: 'ang-my-collections',
   templateUrl: 'my-collections.component.pug',
-  styleUrls: [ './browse-page.component.scss' ]
+  styleUrls: ['./browse-page.component.scss', './tag/tag.component.scss']
```

Comment:
Are you being lazy here?

Reply:
yes :) will bring the styles into the component

Reply:
@codiak SCSS are now added to browse page styles